configure.ac

```diff
@@ -2770,6 +2770,12 @@ if test X"$OPT_LIBMETALINK" != Xno; then
       AC_MSG_NOTICE([libmetalink library defective or too old])
       want_metalink="no"
     ])
+    if test "x$OPENSSL_ENABLED" != "x1" -a "x$USE_WINDOWS_SSPI" != "x1" \
+        -a "x$GNUTLS_ENABLED" != "x1" -a "x$MBEDTLS_ENABLED" != "x1" \
+        -a "x$NSS_ENABLED" != "x1" -a "x$SECURETRANSPORT_ENABLED" != "x1"; then
+      AC_MSG_NOTICE([metalink support requires a compatible SSL/TLS backend])
```

Comment:
Can I suggest using `AC_MSG_WARN` instead of `AC_MSG_NOTICE`?

Reply:
Sure - I'm still a notice at autoconf stuff so this was a real hit in the dark :(